modules/features/manatime_feature_timesheets/manatime_feature_timesheets.module

```diff
@@ -264,6 +273,30 @@ function manatime_feature_timesheets_add_row_form($form, &$form_state) {
 }
 
 /**
+ * Add a new row in the current timesheet.
+ */
+function manatime_feature_timesheets_add_time_entry_form($form, &$form_state) {
```

Comment:
What happens if you define $node as third parameter? Do you receive it? If yes, please implement this way for all of the parameters.